lib/stateMachine.js

```diff
@@ -209,3 +210,30 @@ StateMachine.prototype.map = function map() {
   this.map = this._iter('map');
   return this.map.apply(this, arguments);
 };
+
+/**
+ * Returns a copy of this state machine
+ *
+ * The function profile can look like:
+ * this.forEach(state1, fn);         // iterates over all paths in state1
+ * this.forEach(state1, state2, fn); // iterates over all paths in state1 or state2
+ * this.forEach(fn);                 // iterates over all paths in all states
+ *
+ * @param {String} [state]
+ * @param {String} [state]
+ * @param {Function} callback
```

Comment:
```suggestion
```

from what i can still tell, this function does not take any arugments, does it? (also the documented code above has no relation to this function